include/range/v3/size.hpp

```diff
@@ -38,17 +38,17 @@ namespace ranges
             {
             private:
                 template<typename R, std::size_t N>
-                static constexpr std::size_t impl_(R const (&)[N], int) noexcept
+                static constexpr std::size_t impl_(R (&)[N], int) noexcept
```

Comment:
I `s/const //g`-ed the entire definition. It obviously makes no difference to this overload, but does maintain a consistent appearance.